fluid/voice.h

```diff
@@ -227,6 +234,7 @@ class Voice
       bool ON() const          { return (status == FLUID_VOICE_ON) && (volenv_section < FLUID_VOICE_ENVRELEASE); }
       int SAMPLEMODE() const   { return ((int)gen[GEN_SAMPLEMODE].val); }
 
+      void calcVolEnv(int n,int restN,fluid_env_data_t *env_data);
```

Comment:
Style, should be `void calcVolEnv(int n, int restN, fluid_env_data_t* env_data);`